src/compiler/tsconfig.json

```diff
@@ -1,13 +1,5 @@
 {
-  "compilerOptions": {
-    "allowJs": true,
-    "checkJs": true,
-    "noEmit": true,
-    "paths": {
-      "#compiler": ["./"],
-      "#compiler/*": ["./*"]
-    }
-  },
-  "include": ["**/*.js", "**/*.ts"],
-  "exclude": ["node_modules"]
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {"baseUrl": "../.."},
```

Comment:
FMI: is this necessary in everything that extends it? The `baseUrl` doesn't automatically get updates during extension?

Reply:
Actually, I think you're right; can remove this from all, nice!